js/helpers/misc.js

```diff
@@ -21,7 +21,42 @@ var isValidColor = function(color) {
 	}
 };
 
+// Get value
+var hasArrayKey = function(data, keys) {
+	for (var i = 0; i < keys.length; i++) {
```

Comment:
Why not a forEach? Would read better and reduce lines

Reply:
Because then I would have to polyfill